src/frontends/tensorflow/src/helper_transforms/unique_replacer.hpp

```diff
@@ -0,0 +1,29 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <memory>
+#include <utility>
+
+#include "openvino/frontend/tensorflow/visibility.hpp"
+#include "openvino/pass/graph_rewrite.hpp"
+#include "openvino/pass/pass.hpp"
+
+namespace ov {
+namespace frontend {
+namespace tensorflow {
+namespace pass {
+
+// This transformation expresses Unique with a sub-graph of OpenVINO operations
+class TENSORFLOW_API UniqueReplacer : public ov::pass::MatcherPass {
```

Comment:
why do we still export internal API? I supposed we agreed to update the list of files used in tests https://github.com/openvinotoolkit/openvino/blob/master/src/frontends/tensorflow/src/CMakeLists.txt#L24-L29

Reply:
Agree. That's my bad. I will clean this up.

Reply:
Resolved in https://github.com/openvinotoolkit/openvino/pull/13484